javascript/nanoexpress/config.yaml

```diff
@@ -0,0 +1,12 @@
+framework:
+  github: nanoexpress/nanoexpress
+  version: 2.4.4
+
+environment:
+  EXPERIMENTAL_FASTCALL: 1
```

Comment:
Is this production ready with uWebSocket ?

Reply:
Yes, it is production ready. With this env variable performance boost sometimes improves by 20% which is huge for me